commands/load_balancers.go

```diff
@@ -201,6 +204,28 @@ func RunLoadBalancerCreate(c *CmdConfig) error {
 		return err
 	}
 
+	wait, err := c.Doit.GetBool(c.NS, doctl.ArgCommandWait)
+	if err != nil {
+		return err
+	}
+
+	if wait {
+		lbs := c.LoadBalancers()
+		notice("Load balancer creation is in progress, waiting for load balancer to become active")
+
+		err := waitForActiveLoadBalancer(lbs, lb.ID)
+		if err != nil {
+			return fmt.Errorf(
+				"load balancer couldn't enter `active` state: %v",
+				err,
+			)
+		}
+
+		lb, _ = lbs.Get(lb.ID)
+	}
+
+	notice("Load balancer created")
```

Comment:
This line is causing the test failures.  We could remove it or adjust the expected output:

https://github.com/digitalocean/doctl/blob/dc8385a75cf9062b7de5f3a68c22aa780dbb31fb/integration/lb_create_test.go#L103

Reply:
Which one will you prefer? IMO we should add the notice to make it consistent with the other implementation but I can also get rid of it, just need to know which one makes more sense from the maintainer POV.

Reply:
:+1:  Good point. Let's leave it in.